### DISCUSSION TITLE
Setting up / installing Sunbird ED on local

### DISCUSSION THREAD

**Discussion by vaivk369** [Sunbird-Ed/Community#567] [Installation] [answered]
Hi,
 @rajeevsathish @rajeshkumaravel 
With reference to the issue we are facing here https://github.com/orgs/Sunbird-Knowlg/discussions/142/,  to debug the issue further we need to set up the Subird Ed on local. 
I have followed https://github.com/Sunbird-Ed/SunbirdEd-portal/tree/release-6.0.0 instructions to set up the portal, but it is not working. 
Attached screenshot of the error.
Please help as this is kind off blocker. 
![Screenshot from 2023-06-28 22-39-21](https://github.com/Sunbird-Ed/Community/assets/25919877/8733b7b0-2fc7-4d3c-a890-8b595ebadc63)

@vinukumar-vs @pallakartheekreddy @vaibhavbhuva

**Comment by Krishnaj20** [comment]
@rajeevsathish - This thread has not yet been responded. Request you to look into this or tag the right people and follow up so that timely response is provided.

@chitranshu-keshav @vinukumar-vs

**Reply by Krishnaj20** [reply]
@vaivk369 - is your query addressed?

**Reply by rajeevsathish** [reply, marked as answer]
@vaivk369 Team is working on this we have a jira ticket 
https://project-sunbird.atlassian.net/browse/ED-2616 we will update on the ETA once the root cause is determined and will update

**Reply by Krishnaj20** [reply]
@rajeevsathish - any update on the ETA?

**Reply by rajeevsathish** [reply]
@Krishnaj20 team is busy on other items we may need a week's time on this

**Comment by ajgemini** [comment]
> Hi, @rajeevsathish @rajeshkumaravel With reference to the issue we are facing here https://github.com/orgs/Sunbird-Knowlg/discussions/142/, to debug the issue further we need to set up the Subird Ed on local. I have followed https://github.com/Sunbird-Ed/SunbirdEd-portal/tree/release-6.0.0 instructions to set up the portal, but it is not working. Attached screenshot of the error. Please help as this is kind off blocker. ![Screenshot from 2023-06-28 22-39-21](https://private-user-images.githubusercontent.com/25919877/251043531-8733b7b0-2fc7-4d3c-a890-8b595ebadc63.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTA2NjM1NTMsIm5iZiI6MTc1MDY2MzI1MywicGF0aCI6Ii8yNTkxOTg3Ny8yNTEwNDM1MzEtODczM2I3YjAtMmZjNy00ZDNjLWE4OTAtOGI1OTVlYmFkYzYzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA2MjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNjIzVDA3MjA1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM0YWI2MjYwNTg0Y2FiZTkzYjZjYTI5NDBmNWMwZjc0NzEyMDVkODYxMzlkOTE5NTdiYzE0Y2IyYzU5YTBiNzImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.NMHVyw6LwwO7e_WPL9oSSQ_ZvPk_fVsXstvuvFQUK0s)
> 
> @vinukumar-vs @pallakartheekreddy @vaibhavbhuva

Hi, How can we login as a user in local development ? I need to setup the Sunbird Ed project, although staging environment is not accessible. I am attaching some pictures about my current progress, It will be immense help if you can guide me on the setup
![frontend UI](https://github.com/user-attachments/assets/c8ed53f3-d622-4d6e-92de-e4c2b21c952a)

When I try to go to sunbird staging env, I get this Dns issue, it maybe that sunbird has officially pull down the staging server?
![dns issue](https://github.com/user-attachments/assets/0bb76671-06cd-4000-ba99-84e9d9328642)
